CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+## 0.4.0
+
+* Core changes
+  * [Deprecated] Deprecation of Sawtooth connector
```

Comment:
Guess you have more than just deprecated them now :-)

Reply:
true 😂  - have put out feelers on the RC channels ... will hopefully gather some interest 🤷

Reply:
no replies on the channels :(